homeassistant/components/media_player/samsungtv.py

```diff
@@ -84,7 +88,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 class SamsungTVDevice(MediaPlayerDevice):
     """Representation of a Samsung TV."""
 
-    def __init__(self, host, port, name, timeout, mac):
+    def __init__(self, host, port, name, timeout, mac, uuid=None):
```

Comment:
`def __init__(self, host, port, name, timeout, mac, uuid):` I see no reason make it optional.